.github/workflows/benchmark-artifact-mac.yml

```diff
@@ -72,10 +78,7 @@ jobs:
     needs: build-libraries
     runs-on: ubuntu-latest
     
-    steps: 
-      # - name: Setup tmate session
-      #   uses: mxschmitt/action-tmate@v3.1
-    
+    steps:
```

Comment:
Not sure we want a pre-release.  There should probably be another action for pre-releases.